### DISCUSSION TITLE
User verification, bots and fake profile prevention, small fee for sms verification

### DISCUSSION THREAD

**Discussion by conaman551** [PullDating/Org#206] [Ideas & Features]
As we all know bots and fake profiles are a big issue in dating apps. paying $3 to be sms verified would clear up a lot of problems
although its still optional.

Verifed and Unverified should still have exactly the same exposure though (as per the open source code) Just that verified gets a little badge, and unverified (when app is big) can have a little warning sign for extra vigilance

Can also add in search filter a check for "show verified users only"

**Comment by ajbott** [comment]
**Overview**:
This feature introduces an optional SMS verification process coupled with a nominal $3 fee, aimed at bolstering user authenticity and diminishing the presence of bots and fake profiles within the dating app.

**Benefits for Verified Users**:

- Acquisition of a "Verified" badge on their profile, symbolizing authenticity.
- Inclusion in a special category accessible via a "Verified Only" search filter, enhancing discoverability among users seeking verified profiles.

**Note for Unverified Users**:

As the app expands, profiles lacking verification may feature a cautionary symbol to prompt heightened vigilance among other users. Importantly, the app guarantees equal exposure for both verified and unverified users to ensure equitable visibility within the community.

**Search Customization**:

- A dedicated search filter will be available, permitting users to exclusively browse through verified profiles, should they prefer.

Overall, I think this might be a great strategy to limit the amount of bots on the platform as each bot would have to pay the fee to become verified. I also think that we could still look into non-monetary ways to have the user become verified as well. Maybe if a few verified friends verify the profile they could get in as well? This would promote people to talk about the app and help people get onboard without having to pay.
